### PR TITLE
enable windows builds

### PR DESCRIPTION
**Pull request by msarahan**
Postgres win builds arrived in https://github.com/conda-forge/postgresql-feedstock/pull/2 - libpq in https://github.com/conda-forge/libpq-feedstock/pull/1 - we should be good to go here.

### CODE REVIEW

**Comment by msarahan** [comment]
Very close.  This looks like a pretty simple patch - psycopg2 just defines some things manually that newer VS has actually started to provide.  I'll work on this tomorrow.

**Comment by msarahan** [comment]
@ocefpaf - all green here.  Please review and merge.  The thing that needed patching had already been patched quite a while ago, but the 2.6.2 release is pretty recent (beginning of July).

File: recipe/have_openssl.patch [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
- # Set to 1 if the PostgreSQL library was built with OpenSSL.
- # Required to link in OpenSSL libraries and dependencies.
--have_ssl=0
-+have_ssl=1
+@@ -1,7 +1,7 @@
+ [build_ext]
+ define = 
+ use_pydatetime = 1
+-have_ssl = 0
++have_ssl = 1
```
Comment (ocefpaf) [inline recipe/have_openssl.patch]: :+1:

**Comment by ocefpaf** [comment]
Awesome! Thanks @msarahan!!

**Pull request merged**